docs/buildah-build.1.md

```diff
@@ -300,19 +300,19 @@ solely for scripting compatibility.
 
 **--dns**=[]
 
-Set custom DNS servers
+Set custom DNS servers.  Invalid if using --dns with --network that is set to none.
```

Comment:
LGTM, but am curious, why the spelling-out? Wouldn't `...with --network=none` be a little clearer?

Reply:
Cutting and pasting from the podman man pages.